main.go

```diff
@@ -59,6 +62,28 @@ func main() {
 	// Initialize the database handler
 	pizzaOven := database.NewPizzaOvenDbHandler(databaseHost, databasePort, databaseUser, databasePwd, databaseDbName)
 
+	// Initializes configuration using a provided yaml file
+	config := &server.Config{NeverEvictRepos: make(map[string]int)}
+	var configParser struct {
+		NeverEvictRepos []string `yaml:"never-evict-repos"`
+	}
+
+	configFile, err := os.ReadFile(configPath)
+	if err != nil {
+		sugarLogger.Debugf("No yaml configuration file was provided")
```

Comment:
This chunk will mask any error from `os.ReadFile` and make debugging missing or mis-configured yaml file paths difficult. Can the flow instead be:

```
if config flag is not empty:
    read file using ReadFile and get error:
        if error is not empty, return error